composer/algorithms/cutout/README.md

```diff
@@ -42,13 +42,23 @@ def training_loop(model, train_loader):
 
 ### Composer Trainer
 
-<!-- TODO: Address timeouts -->
-<!--pytest-codeblocks:skip-->
+<!--pytest.mark.gpu-->
+<!--pytest.mark.timeout(15)-->
+<!--
+```python
+from torch.utils.data import DataLoader
```

Comment:
I see this test fixture setup duplicated in a few places...does this plugin support pytest fixtures? Is there a way to reduce the duplicative code? If it'd make things harder to read/follow then no worries -- fine to leave it as is.

Reply:
Unfortunately no :(. It's definitely possible to extend the base repo to support it, but I don't think its worth the time / effort since its just cleanliness vs. functionality

Reply:
OK, agreed.